azure-kusto-data/source/connectionBuilder.js

```diff
@@ -39,6 +39,16 @@ const KeywordMapping = Object.freeze({
         mappedTo: "Authority Id",
         validNames: ["authority id", "authorityid", "authority", "tenantid", "tenant", "tid"]
     },
+    msi_endpoint: {
```

Comment:
`msi_enpoint` and `msi_secret` are neither documented as part of a query string nor I think they exists other than on this PR. I decided to add it here to not change a lot the architecture of the application.

Feel free to comment if you have a better idea for how to handle this.

Reply:
camelCase instead of snake_case?

Reply:
I have changed the variable names as suggested. I leaved the snake_case and add an all lowercase on the validNames as that's the name the documentation used for Managed Identities. 
Let me know if you still wanted to change it.